app/assets/stylesheets/conversation_show.scss

```diff
@@ -96,25 +100,26 @@ margin: 0 0 5px 0;
   hyphens: auto;
 }
 
-.fixed-height-container{
-  @include shadow;
-  height: 50%;
-  width: 90%;
-  margin: auto;
-  padding:3px;
-  // background:#666;
-  // margin: 0 0 2px 10px;
-}
+// .fixed-height-container{
+//   // @include shadow;
+//   height: 60%;
+//   width: 100%;
+//   margin: auto;
+//   padding:3px;
+//   // background:#666;
+//   // margin: 0 0 2px 10px;
+// }
 
 .chatboxcontent {
-  height:95%;
+  height:60%;
```

Comment:
Properties should be ordered background, height, line-height, list-style, overflow<br>Colon after property should be followed by one space